addons/mail/static/src/rtc/rtc_service.js

```diff
@@ -1417,11 +1422,7 @@ export class Rtc {
         stream.addTrack(track);
         if (track.kind === "audio") {
             const audioElement = session.audioElement || new window.Audio();
-            try {
```

Comment:
removal of try/catch explained in commit message, in addition to the message browser versions that do not support `srcObject` are very unlikely to support webRTC anyways.